src/Components/RecommendeeCard/RecommendeeCard.js

```diff
@@ -1,9 +1,48 @@
 import './_RecommendeeCard.scss'
 
-const RecommendeeCard = () => {
-    return(
-        <div className="recommendee-card-container">
-            <h1>RecommendeeCard</h1>
+const RecommendeeCard = ({ poster, title, releaseYear, rating, genres, date }) => {
+    const allGenres = genres.reduce((genreCategories, currentGenre) => {
+        genreCategories += currentGenre
+        if (currentGenre != genres[genres.length - 1]) {
```

Comment:
This is for line 6.  !== will do a more strict compare than !=. !== will check if the values being compared are the same type as well as the same value. Here is a StackOverflow that has a good example for why !== is important: https://stackoverflow.com/questions/8616483/javascript-comparison-operators-vs

Reply:
ty!